src/components/Workshop/WorkshopCard.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/Constants.scss';
+
+#workshop-background {
+    position:relative;
+    width: 331px;
+    height: 452px;
+    background-color: grey;
+    background-blend-mode: multiply;
+}
+
+#workshop-content {
+    position:absolute;
+    top:61%;
+    color:$neutral-color-95;
+    margin-left:20px;
+    width:50%;
```

Comment:
Niespecjalnie mi się to podoba, nie lepiej wrzucić padding na cały kontener? 

Na dodatek w wypadku gdy szkolenie będzie miało długą nazwę, to wszystko się wykrzacza. Bład nie widnieje tylko w tym miejscu ale w całym komponencie 

![image](https://user-images.githubusercontent.com/84678175/216839299-4d4cdf9a-7390-41f5-a6e4-bc9d42e04a07.png)

Reply:
Testowałem to dając obrazek jako background. Na początku padding na kontenerze przesuwał to szare tło i wystawało poza obrazek. Teraz jak zostały zrobione zmiany to padding działa dobrze.